packages/ses/src/module-instance.js

```diff
@@ -29,6 +29,8 @@ export const makeModuleInstance = (
 
   const compartmentFields = privateFields.get(compartment);
 
+  const { __shimTransforms__ } = compartmentFields;
```

Comment:
I am surprised that `compartmentFields` is a transform?

Reply:
The shim transforms are an array destructured from the compartment fields object.